components/search/SearchForm.vue

```diff
@@ -230,11 +230,8 @@
 
         this.gettingSuggestions = true;
 
-        // Query in the user's language, and English, removing duplicates
-        const languageParam = Array.from(new Set([this.$i18n.locale, 'en'])).join(',');
-
-        const suggestions = await getEntitySuggestions(query, {
-          language: languageParam
+        const suggestions = await getEntitySuggestions(encodeURIComponent(query), {
```

Comment:
Do not URI-encode this. Instead, catch any errors from the API response and just show no suggestions.